tests/sympc/tensor/mpc_tensor_test.py

```diff
@@ -250,6 +250,27 @@ def test_ops_public_mpc(get_clients, nr_clients, op_str) -> None:
     assert np.allclose(result, expected_result, atol=10e-4)
 
 
+falcon = Protocol.registered_protocols["Falcon"]()
+
+
+@pytest.mark.parametrize("op_str", ["add", "sub"])
+def test_ops_public_mpc_falcon(get_clients, op_str) -> None:
+    clients = get_clients(3)
+    session = Session(parties=clients, protocol=falcon)
+    SessionManager.setup_mpc(session)
+
+    op = getattr(operator, op_str)
+
+    x_secret = torch.Tensor([[0.125, -1.25], [-4.25, 4]])
+    y_secret = torch.Tensor([[4.5, -2.5], [5, 2.25]])
```

Comment:
I know this isn't a big deal, but still a little misleading. Could you name y_secret as public_val or just y?

Reply:
Thats a good suggestion,I would modify :+1: